packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -231,7 +235,9 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
    */
   private sendAnnouncement = (): void => {
     if (this.hasVoiceaJoined || !this.webex.internal.llm.isConnected()) return;
-    this.listenToEvents();
+    if (!this.hasSubscribedToEvents) {
```

Comment:
it's better to check that flag inside listenToEvents(), so that in the future if anyone adds another call to this.listenToEvents() the flag will be already checked for them.

Reply:
Modified